todolist/app.go

```diff
@@ -193,6 +193,36 @@ func (a *App) getId(input string) (int, *Todo) {
 	}
 }
 
+func (a *App) getIds(input string) []int {
+	var ids []int
+	idGroups := strings.Split(input, ",")
+
+	singleNumberRE, _ := regexp.Compile("\\d+")
+	rangeNumberRE, _ := regexp.Compile("(\\d+)-(\\d+)")
+
+	for _, idGroup := range idGroups {
+		if matches := rangeNumberRE.FindStringSubmatch(idGroup); len(matches) > 0 {
+			lowerID, _ := strconv.Atoi(matches[1])
+			upperID, _ := strconv.Atoi(matches[2])
+			if lowerID >= upperID {
+				fmt.Printf("Invalid id group: %s.\n", idGroup)
+				break
+			}
+			for id := lowerID; id <= upperID; id++ {
+				ids = append(ids, id)
+			}
```

Comment:
Ok I have update my PR, it might not be what you were expecting.

I am happy to revert these changes are they are a little deeper that what you may have wanted.